Nos/Views/AuthorStoryView.swift

```diff
@@ -28,67 +34,191 @@ struct AuthorStoryView: View {
     }
     
     var body: some View {
-        VStack {
-            ScrollView {
-                if let note = notes[safe: noteIndex] {
-                    NoteButton(
-                        note: note,
-                        showFullMessage: true,
-                        hideOutOfNetwork: false
-                    ) 
-                    .id(noteIndex) // TODO: Why doesn't it work without this!?
-                    .allowsHitTesting(false)
-                    .padding()
+        ScrollView(.vertical, showsIndicators: false) {
+            Group {
+                if let selectedNote {
+                    CompactNoteView(note: selectedNote, showFullMessage: false)
+                        .padding(.top, 60)
                 } else {
-                    Text("empty")
+                    EmptyView()
                 }
             }
-            Spacer()
-            HStack {
-                if let currentNote = notes[safe: noteIndex] {
-                    ForEach(notes.indices) { noteIndex in
-                        RoundedRectangle(cornerRadius: 21)
-                            .frame(maxWidth: .infinity, maxHeight: 3)
-                            .padding(1.5)
-                            .cornerRadius(21)
-                            .foregroundColor(noteIndex <= self.noteIndex ? .accent : .secondaryText)
+        }
+        .id(selectedNote)
+        .overlay(alignment: .leading) {
+            Button {
+                guard let selectedNote, let selectedNoteIndex = notes.firstIndex(of: selectedNote) else {
+                    return
+                }
+                if selectedNoteIndex > 0 {
+                    let previousIndex = notes.index(before: selectedNoteIndex)
+                    self.selectedNote = notes[safe: previousIndex]
+                } else {
+                    showPreviousAuthor()
+                }
+            } label: {
+                Color.red.opacity(0)
```

Comment:
This sounds weird, but it enables us to define tappable regions that overlap the underlying tappable regions of the displayed note.